packages/mask/src/plugins/Furucombo/base.tsx

```diff
@@ -15,5 +15,6 @@ export const base: Plugin.Shared.Definition = {
     },
     contribution: {
         postContent: new Set([/https:\/\/furucombo.app\/invest\/(pool|farm)\/(137|1)\/(0x\w+)/]),
+        metadataKeys: new Set([PLUGIN_ID]),
```

Comment:
This is strange. Because this plugin does not contribute to metadata.

Reply:
fixed.